### DISCUSSION TITLE
row and columns with 1 element

### DISCUSSION THREAD

**Discussion by tianhuat** [quasarframework/quasar#11301] [General - Components / Directives / etc] [answered]
For the example given here:
https://quasar.dev/layout/grid/gutter#pros-cons-and-how-to-workaround-problems-q-gutter-size-vs-q-col-gutter-size
May I know
1. What's the functions of row and column in the example as we only have 1 item within?
2. Why using row and overflow-auto can solve the gutter-size issue?

[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/oXN0S.png

**Comment by pdanpdan** [comment, marked as answer]
1.
- first `row` - to keep the 3 examples in line
- first `column` - you have a title (`p`) and the `div` with example
- second `row` - you have 4 elements with `v-for`

2. `overflow: auto` is in some way opposed to `overflow: visible` - it does not allow the content to show outside, but shows scroll if required
